src/PowerShellEditorServices/Extensions/BuiltInCommands.ps1

```diff
@@ -0,0 +1,29 @@
+Register-EditorCommand `
+    -Name 'PowerShellEditorServices.OpenEditorProfile' `
+    -DisplayName 'Open Editor Profile' `
+    -SuppressOutput `
+    -ScriptBlock {
+        param([Microsoft.PowerShell.EditorServices.Extensions.EditorContext]$context)
+        If (!(Test-Path -Path $Profile)) { New-Item -Path $Profile -ItemType File }
+        $psEditor.Workspace.OpenFile($Profile)
+    }
+
+Register-EditorCommand `
+    -Name 'PowerShellEditorServices.OpenProfileList' `
+    -DisplayName 'Open Profile from List (Current User)' `
+    -SuppressOutput `
+    -ScriptBlock {
+        param([Microsoft.PowerShell.EditorServices.Extensions.EditorContext]$context)
+        
+        $List = @('Microsoft.VSCode_profile.ps1','Microsoft.PowerShell_profile.ps1','Microsoft.PowerShellISE_profile.ps1','Profile.ps1')
```

Comment:
If you want to wait and make this change later, that's alright.  If you want to make it now you can just make an additional commit in your branch and push it to your repo.  The PR should pick up the change automatically.